### ISSUE TITLE
Un-initialized value of DateFormula column?

### ISSUE THREAD

**Issue by wkaper1** [MicrosoftDocs/dynamics365smb-devitpro-pb#1152]
I think this page should state how one can recognize a DateFomula column value that has not been provided by the user.
Example: Column ShippingTime in table SalesHeader is of type DateFormula, now how do I recognize an order where this column is not filled? It turns out this test does it:
if FORMAT(DateFormulaVariable) = ''  //nullstring, not space
A useful piece of information!
What 's the use of comparing a Dateformula to a space ' '? May be it has some use but you forgot tot tell. Or you could replace the given example by a more useful one. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9b659a96-ac9d-4c56-4e7d-b85abc8eb465
* Version Independent ID: 050cf5d3-4c38-4b3a-cc0f-e56891d7249b
* Content: [DateFormula Data Type - Business Central](https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/dateformula/dateformula-data-type#feedback)
* Content Source: [dev-itpro/developer/methods-auto/dateformula/dateformula-data-type.md](https://github.com/MicrosoftDocs/dynamics365smb-devitpro-pb/blob/live/dev-itpro/developer/methods-auto/dateformula/dateformula-data-type.md)
* Service: **dynamics365-business-central**
* GitHub Login: @SusanneWindfeldPedersen
* Microsoft Alias: **solsen**

**Comment by db-bc** [comment]
> how do I recognize an order where this column is not filled? It turns out this test does it:
> if FORMAT(DateFormulaVariable) = '' //nullstring, not space

Right, the page's example comparing to a space still seems wrong/unnecessary.

I prefer to instead declare a local `var BlankDateFormula: DateFormula;`, never assign to it, and compare to that: `if DateFormulaVariable = BlankDateFormula`. Then no time wasted doing `Format()`.

But it would be preferable to have a method like `DateFormulaVariable.IsBlank()` or `.IsNull()` or such (could be a free function instead of a member, like `IsNullGuid()`. Having to declare a blank variable is more boilerplate that ideally wouldn't be needed.
